<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QDialog" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>SQL:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::TextEditorInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QComboBox" name="poleCombo">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Землепользователь</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>СМО</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Тип поля</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номер поля</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Площадь поля</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Культура</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Номер раб. уч.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Площадь раб. уч.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Площадь почв. конт.</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Индекс почв</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Гранулометрический состав</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Засоренность</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Гамма-фон</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>N</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>P</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Org</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>S</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ph</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Ca</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Na</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>РЕКО</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cd</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>As</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hg</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Zn</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>B</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="exprCombo">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&gt;=</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>&lt;=</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="valueCombo">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="exprButton">
       <property name="text">
        <string>Add expression</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="separCombo">
       <item>
        <property name="text">
         <string>AND</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OR</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="separButton">
       <property name="text">
        <string>Add separator</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
